pynestml/codegeneration/resources_nest/point_neuron/directives/IfStatement.jinja2

```diff
@@ -3,14 +3,14 @@
   @param ast ASTIfStmt
 #}
 {%- if tracing %}/* generated by {{self._TemplateReference__context.name}} */ {% endif %}
-if ({{printer.print_expression(ast.get_if_clause().get_condition())}})
+if ({{printer.print(ast.get_if_clause().get_condition())}})
```

Comment:
Should we do this in other Jinja templates as well?

Reply:
done!